lib/fennec/turn.ex

```diff
@@ -9,6 +9,7 @@ defmodule Fennec.TURN do
     nonce: nil, realm: nil
 
   @permission_lifetime 5 * 60 # MUST be 5mins
+  @channel_lifetime 10 * 60 # 10 minutes
```

Comment:
Is that the requirement or suggestion according to the standard? Maybe it's better to clarify in order not to force the reader to google the RFC :)

Reply:
It's a requirement, will fix that.